lib/encoder.py

```diff
@@ -766,7 +770,11 @@ def _FilesToEncodings(self, files, videofile, bitrate=None,
         encoder or self._FileNameToEncoder(full_filename),
         bitrate or _FileNameToBitrate(full_filename),
         videofile or _FileNameToVideofile(full_filename))
-      candidate.Recover()
+      try:
+        candidate.Recover()
+      except Error:
+        print 'Ignoring error in file %s' % full_filename
```

Comment:
Yep. Except that "print" is bad. Perhaps the disk cache should have an auxillary "self" attribute with the filenames that are known to be bad and their error messages?